File: Python/adyen.py
import Adyen

def main():
    adyen = Adyen.Adyen()
    adyen.payment.client.xapikey = "YOUR_API_KEY"
    adyen.payment.client.platform = "test"  # Change the value to live for the live environment.

    request = {
        "amount": {
            "currency": "USD",
            "value": 1000 # Value in minor units.
        },
        "reference": "My first Adyen test payment with an API library/SDK",
        "paymentMethod": {
            "type": "visa",
            "encryptedCardNumber": "test_4111111111111111",
            "encryptedExpiryMonth": "test_03",
            "encryptedExpiryYear": "test_2030",
            "encryptedSecurityCode": "test_737"
        },
        "shopperReference": "YOUR_SHOPPER_REFERENCE",
        "returnUrl": "https://your-company.com/...",
        "merchantAccount": "YOUR_MERCHANT_ACCOUNT"
        }

    result = adyen.checkout.payments_api.payments(request, idempotency_key="YOUR_IDEMPOTENCY_KEY")

File: proj dotnet/Dotnet/Adyen.cs
using Adyen;
using Adyen.Model.Checkout;
using Adyen.Service;
using Environment = Adyen.Model.Environment;


namespace AdyenExample
{
    class Program
    {
        static void Main(string[] args)
        {
// Create a paymentRequest object.
var amount = new Amount("USD", 1000);
var paymentRequest = new PaymentRequest
{
    Reference = "My first Adyen test payment with an API library/SDK",
    Amount = amount,
    ReturnUrl = @"https://your-company.com/...",
    MerchantAccount = "YOUR_MERCHANT_ACCOUNT",
};

// Set up the client and service.
var config = new Config
{
    XApiKey = "YOUR_API_KEY",
    Environment = Environment.Test
};
var client = new Client(config);
var checkout = new PaymentsService(client);

// Add your idempotency key.
var requestOptions = new RequestOptions { IdempotencyKey= "YOUR_IDEMPOTENCY_KEY" };

// Make a /payments request.
var paymentResponse = checkout.Payments(paymentRequest, requestOptions);
        }}}